images/php-cli/8.3.Dockerfile

```
@@ -14,7 +14,7 @@ RUN apk add --no-cache git \
         mariadb-client \
         mariadb-connector-c \
         mongodb-tools \
-        nodejs-current=~20 \
+        nodejs-current=~21 \
```

```suggestion
        nodejs=~20 \
```
We should avoid changing the version here for users - especially to a non-LTS release

---

Makes 100% sense.